inc/class-onboarding.php

```diff
@@ -0,0 +1,569 @@
+		$this->recommended_plugin_files();
+
+		$file    = 'elementor/elementor.php';
+		$plugins = get_plugins();
+
+		if ( ! in_array( $file, array_keys( $plugins ), true ) ) {
+			$slug      = 'elementor';
+			$installer = new \Plugin_Upgrader( new \Automatic_Upgrader_Skin() );
+
+			$api = plugins_api(
+				'plugin_information',
+				array(
+					'slug'   => $slug,
+					'fields' => array(
+						'sections' => false,
+					),
+				)
+			);
+
+			if ( is_wp_error( $api ) ) {
+				return false;
+			}
+
+			$results = $installer->install(
+				$api->download_link,
+			);
+
+			if ( is_wp_error( $results ) || ! $results ) {
+				return false;
+			}
+		}
+
+		$results = activate_plugin( $file );
+
+		if ( is_wp_error( $results ) ) {
+			return false;
+		}
+
+		return true;
+	}
+
+	/**
+	 * Install and activate Elementor Pro plugin.
+	 *
+	 * @return bool
+	 */
+	private function install_hello_elementor() {
+		// Include required files.
+		$this->recommended_files();
+		$this->recommended_theme_files();
+
+		$themes     = wp_get_themes();
+		$stylesheet = 'hello-elementor';
+
+		if ( ! in_array( $stylesheet, array_keys( $themes ), true ) ) {
+			$installer = new \Theme_Upgrader( new \Automatic_Upgrader_Skin() );
+
+			$api = themes_api(
+				'theme_information',
+				array(
+					'slug'   => $stylesheet,
+					'fields' => array(
+						'sections' => false,
+					),
+				)
+			);
+
+			if ( is_wp_error( $api ) ) {
+				return false;
+			}
+
+			$results = $installer->install(
+				$api->download_link,
+			);
+
+			if ( is_wp_error( $results ) || ! $results ) {
+				return false;
+			}
+		}
+
+		// Switch theme.
+		switch_theme( 'hello-elementor' );
+
+		return true;
+	}
+
+	/**
+	 * Enables Elementor Container Experiment.
+	 *
+	 * @return bool
+	 */
+	private function enable_el_container_experiment() {
+		if ( ! did_action( 'elementor/loaded' ) ) {
+			return false;
+		}
+
+		$key = 'elementor_experiment-container';
+
+		if ( 'active' === get_option( $key ) ) {
+			return true;
+		}
+
+		$result = update_option( $key, 'active' );
+
+		if ( ! $result ) {
+			return false;
+		}
+
+		return true;
+	}
+
+	/**
+	 * Enables Elementor Container Experiment.
+	 *
+	 * @return bool
+	 */
+	private function disable_el_defaults() {
+		$color_key = 'elementor_disable_color_schemes';
+		$fonts_key = 'elementor_disable_typography_schemes';
+
+		if ( 'yes' === get_option( $color_key ) && 'yes' === get_option( $fonts_key ) ) {
+			return true;
+		}
+
+		$color_schemes = update_option( $color_key, 'yes' );
+		$font_schemes  = update_option( $fonts_key, 'yes' );
+
+		if ( ! $color_schemes && ! $font_schemes ) {
+			return false;
+		}
+		return true;
+	}
+
+	/**
+	 * Imports base style kit.
+	 *
+	 * @return bool
+	 */
+	private function import_base_kit() {
+		$kit = array(
+			'id'             => 6491,
+			'title'          => 'Style Kit: Base',
+			'image'          => 'https://preview.analogwp.com/designsystem/wp-content/uploads/sites/26/2022/11/base.svg',
```

Comment:
Lets not add a remote image here, since that can be removed at any point.

Should be saved locally.

Reply:
Resolving this, as it is already being done at import.